client/src/components/GameBadge.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import '../containers/App.css';
+
+
+export default class GameBadge extends Component {
+
+    render() {
+        const link="http://megaminerAI.com/" + this.props.gameID;
+				let bgColor = ''
+				// The Background Color of the div should be different based on what the result of the game
+				switch(this.props.result) {
+					case 'Win':
+						bgColor= '#bef5cb'
+						break
+					case 'Lose':
+						bgColor= '#fdaeb7'
+						break
+					default:
+						bgColor= '#cccccc'
+						break
+				}
+
+        return(
+          <div>
+            <div style={{ backgroundColor:bgColor }} className="gamebadge-wrapper col-lg-12">
+
+              <div className="gamebadge-info left">
```

Comment:
So, Bootstrap has a grid system that you can use, and will probably make more sense then using a custom CSS rule. Check it out [here](https://getbootstrap.com/docs/3.3/css/#grid) and let me know if you have any questions about implementing it.